scene/gui/line_edit.cpp

```diff
@@ -2148,6 +2159,7 @@ void LineEdit::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_secret_character"), &LineEdit::get_secret_character);
 	ClassDB::bind_method(D_METHOD("menu_option", "option"), &LineEdit::menu_option);
 	ClassDB::bind_method(D_METHOD("get_menu"), &LineEdit::get_menu);
+	ClassDB::bind_method(D_METHOD("is_menu_visible"), &LineEdit::is_menu_visible);
```

Comment:
Any particular reason why was this bound? Is it useful outside source? You could just do `get_menu().visible`, I don't think it's that important for scripting.

Reply:
It seems in many places, code gets the menu to check if its visible. Because we don't create it until needed, this forces the creation. 

The new function lets you check this without creating it.

Reply:
Yeah, I get why the function exists. But I don't see why you need to bind it for GDSCript. The users don't check the visibility so often for that to matter.